src/agent0/chainsync/db/hyperdrive/interface.py

```diff
@@ -725,110 +680,42 @@ def get_ticker(
     DataFrame
         A DataFrame that consists of the queried pool info data.
     """
-    # pylint: disable=too-many-arguments
-    query = session.query(Ticker)
+    # TODO do we want to keep pools seperate?
```

Comment:
could add an optional hyperdrive_address argument? or just let the user down-select on their own. I think either is fine... Although I guess if we want to be consistent then we would have two endpoints: one for filtering by user and one for filtering by pool?

Reply:
This is a bit weird for this function because it's aggregating. We actually have 3 options for pools, (1) filter by the wallet address then aggregate, (2) aggregate by pool and return for all pools, (3) aggregate across pools.

I'll keep this function as (3) for the time being, and revisit if a need arises.